server/routes/api/webhookDeliveries.ts

```diff
@@ -0,0 +1,35 @@
+import Router from "koa-router";
+import auth from "@server/middlewares/authentication";
+import { WebhookDelivery } from "@server/models";
+import { authorize } from "@server/policies";
+import { presentWebhookDelivery } from "@server/presenters";
+import pagination from "./middlewares/pagination";
+
+const router = new Router();
+
+router.post(
+  "webhookDeliverys.list",
+  auth({ admin: true }),
+  pagination(),
+  async (ctx) => {
+    const { user } = ctx.state;
+    const { webhookSubscriptionId } = ctx.request.body;
+    authorize(user, "listWebhookDeliveries", user.team);
```

Comment:
Need to change this so that it queries the webhookSubscription and then have the policy run against that, currently you could pass any team's `webhookSubscriptionId` and exfiltrate data.